pkgs/applications/misc/rofi/wrapper.nix

```diff
@@ -0,0 +1,14 @@
+{ symlinkJoin, rofi_unwrapped, makeWrapper, theme ? null, lib }:
+
+symlinkJoin {
```

Comment:
i would use simple ``mkDerivation`` since we only want to link `$out/bin/rofi` in our wrapper and also make it overridable.

```nix
{ stdenv, lib, rofi-unwrapped, makeWrapper, theme ? null }
let
  wrapper = stdenv.mkDerivation {
    name = "rofi-${version}";
    buildInputs = [ makeWrapper ];
    preferLocalBuild = true;
    passthru = { unwrapped = rofi-unwrapped; };
    buildCommand = ''
      ln -s ${rofi-unwrapped}/bin/rofi
      wrapProgram $out/bin/rofi --add-flags "-theme ${theme}"
    '';
    meta = rofi-unwrapped.meta // {
      # prefer wrapper over the package
      priority = (rofi-unwrapped.meta.priority or 0) -1;
    };
  };
in lib.makeOverridable wrapper
```